lib/coxir/struct/user.ex

```diff
@@ -102,6 +102,23 @@ defmodule Coxir.Struct.User do
     |> Channel.pretty
   end
 
+  @doc """
+  Sends a DM to a given user.
+
+  Returns a message object upon success
+  or a map containing error information.
```

Comment:
Can we change this with;

*Refer to `Coxir.Struct.Channel.send_message/2` for more information.*